glslc/README.asciidoc

```diff
@@ -185,13 +196,17 @@ preprocessing errors are written to standard error. If multiple input shader
 files are given, their preprocessed output are all written to standard output,
 in the order specified on the command line.
 
+glslc will do nothing for SPIR-V assembly files with this option.
```

Comment:
Similarly to this, we should mention on line 113 that `-fshader-stage` affects only GLSL files.

Also on line 117, `.spvasm` should be excepted.